README.md

```diff
@@ -1,18 +1,66 @@
-# TYPO3 Extension: walls_io_cache
+# TYPO3 Extension: walls_io_proxy
 
-walls.io is a service to show you entries/posts of a specific topic or hashtag from various social media websites
-like Facebook, Twitter and Instagram on your website. But as long as you don't have a
-Premium Account (500 € each month. Date: 10.01.2020) you can not use their API and you have to accept their cookies
-while using their embedded iframe or JS file implementation. With walls_io_proxy no Cookie of walls.io will be set
-on client-side anymore, as we have moved all XHR/Ajax Requests of their JS file to server-side (PHP).
+[![Packagist][packagist-logo-stable]][extension-packagist-url]
+[![Latest Stable Version][extension-build-shield]][extension-ter-url]
+[![License][LICENSE_BADGE]][extension-packagist-url]
+[![Total Downloads][extension-downloads-badge]][extension-packagist-url]
+[![Monthly Downloads][extension-monthly-downloads]][extension-packagist-url]
+[![TYPO3 12.4][TYPO3-shield]][TYPO3-12-url]
 
-Our walls_io_proxy is no API and will not use the API of walls.io.
+![Build Status](https://github.com/jweiland-net/walls_io_proxy/workflows/CI/badge.svg)
 
-As walls_io_proxy gets all data from walls.io now it's up to you to create your own FluidTemplate for your
-social wall. We don't support all their integrated Themes, Styles and Views. You have the data now, do what you want.
+**walls.io** is a service that displays entries or posts related to a specific topic or hashtag from various social media websites like Facebook, Twitter, and Instagram on your website. However, without a Premium Account (costing €500 per month as of January 10, 2020), you cannot use their API and must accept their cookies when using their embedded iframe or JS file implementation.
 
-# Installation
+## Introduction
 
-You can install this extension via Composer with:
+The **walls_io_proxy** extension for TYPO3 addresses this limitation by eliminating the need for client-side cookies from walls.io. It achieves this by moving all XHR/Ajax requests from walls.io's JavaScript file to the server-side using PHP. This approach ensures that no walls.io cookies are set on the client-side, enhancing privacy and compliance with cookie regulations.
 
-`composer req jweiland/walls-io-proxy`
+Please note, **walls_io_proxy** is not an API and does not use the API of walls.io.
```

Comment:
Does not use the API of walls.io?!?!?!

https://github.com/jweiland-net/walls_io_proxy/blob/main/Classes/Request/PostsRequest.php#L20